commit 1ea5a119bd7af00870c90cca10452a87634687fd (refs/changes/00/5700/3)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2019-03-04T18:45:17+00:00 (7 months ago)
    
    TRITON-1276 sdcadm experimental update-gz-tools --latest --just-download dumps core
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/cli/do_update_gz_tools.js b/lib/cli/do_update_gz_tools.js
index 8baee6f..0cbdbee 100644
--- a/lib/cli/do_update_gz_tools.js
+++ b/lib/cli/do_update_gz_tools.js
@@ -509,7 +509,14 @@ UpdateGzTools.prototype.execute = function cExecute(opts, args, callback) {
                 next(err);
             });
         },
-        steps.servers.ensureServersRunning,
+
+        function ensureRunning(ctx, next) {
+            if (justDownload) {
+                next();
+                return;
+            }
+            steps.servers.ensureServersRunning(ctx, next);
+        },
 
         function downloadTarball(_, next) {
             if (filepath) {
